source/Ship.cpp

```diff
@@ -3103,9 +3103,9 @@ void Ship::ApplyForce(const Point &force, bool gravitational)
 		return;
 	
 	// Reduce acceleration of small ships and increase acceleration of large
-	// ones by having 30% of the force be based on a fixed mass of 400, i.e. the
-	// mass of a typical light warship:
```

Comment:
Huh. The things I never knew because I don't know the engine code. So acceleration isn't purely based on thrust and mass, and actually has an artificial limiter arbitrarily slowing down the fast ships?

Now I'm curious what simply removing this would do. Some of the changes done in both Revamp and Test are based around small ships needing to be a bit faster relevant to big ships.

Reply:
This only applied to acceleration caused by hit force weapons.

Reply:
Ah, that's good. Thanks for the explanation.